gravitee-apim-console-webui/src/management/instances/instance-list/instance-list.component.html

```diff
@@ -0,0 +1,118 @@
+<!--
+
+    Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<div class="title">
+  <h1>Gateways</h1>
+</div>
+
+<gio-table-wrapper [disableSearchInput]="true" [length]="nbTotalInstances" [filters]="filters" (filtersChange)="onFiltersChanged($event)">
+  <table mat-table [dataSource]="filteredTableData" id="instancesTable" aria-label="Gateways table">
+    <!-- Display name Column -->
+    <ng-container matColumnDef="hostname">
+      <th mat-header-cell *matHeaderCellDef id="hostname">Name</th>
+      <td mat-cell *matCellDef="let element">
+        <a [uiSref]="'management.instance.environment'" [uiParams]="{ instanceId: element.id }" title="{{ element.hostname }}">{{
+          element.hostname
+        }}</a>
+      </td>
+    </ng-container>
+
+    <!-- Display version Column -->
+    <ng-container matColumnDef="version">
+      <th mat-header-cell *matHeaderCellDef id="version">Version</th>
+      <td mat-cell *matCellDef="let element">
+        {{ element.version }}
+      </td>
+    </ng-container>
+
+    <!-- Display State Column -->
+    <ng-container matColumnDef="state">
+      <th mat-header-cell *matHeaderCellDef id="state">Status</th>
+      <td mat-cell *matCellDef="let element">
+        <mat-icon
+          *ngIf="element.state === 'STARTED'"
+          matTooltip="Started"
+          class="state__instance-started"
+          size="20"
+          svgIcon="gio:play-circle"
+        ></mat-icon>
+        <mat-icon
+          *ngIf="element.state === 'STOPPED'"
+          matTooltip="Stopped"
+          class="state__instance-stopped"
+          size="20"
+          svgIcon="gio:stop-circle"
+        ></mat-icon>
+        <mat-icon
+          *ngIf="element.state !== 'STOPPED' && element.state !== 'STARTED'"
+          matTooltip="Unknown"
+          class="state__instance-unknown"
+          size="20"
+          svgIcon="gio:question-mark-circle"
+        ></mat-icon>
```

Comment:
Do we have only 3 element.state properties? Could we just use here `element.state === 'UNKNOWN'`?